app.py

```diff
@@ -20,15 +22,28 @@ def get_model(self, num_labels: int):
         )
         return model, tokenizer
```

Comment:
If the user wants to bring their own model which is not a bloom model, would they be able to modify this method and still use the `finetune()` method as is below?

Reply:
Yes